src/charm.py

```diff
@@ -154,11 +156,26 @@ def __init__(self, *args, **kargs) -> None:
             self.unit.status = ops.BlockedStatus(exc.msg)
             return
 
+        self._event_timer = EventTimer(self.unit.name)
+        if not self._event_timer.is_active(RECONCILE_RUNNERS_EVENT):
+            logger.warning(
+                "Reconciliation event timer is not activated - "
+                "this should only happen on the first charm execution after a deployment "
+                "on a new machine"
+            )
+            try:
+                self._set_reconcile_timer()
+            except TimerEnableError as ex:
+                logger.exception("Failed to start the event timer")
+                self.unit.status = BlockedStatus(
```

Comment:
Out of curiosity: is there an alarm/notification for this case?

Reply:
No, not at the moment. One could try to create an alert based on the log. But unfortunately there seems to be a bug in the grafana agent / loki charm that does not maintain the correct juju topology (so only log alerts can be defined for the grafana agent).